.golangci.yml

```diff
@@ -76,6 +76,8 @@ linters-settings:
       - stdlib
       # also allow generics
       - generic
+      # we want to return spi.Interfaces in routing.go
+      - spi.RoutingTemplateLoader
```

Comment:
Are there other loaders we would like to exclude here too?

Reply:
good point, checked for another `//nolint` for ireturn and it looks like in `authconfig.go` we also nolint here since we want to allow `spi.AuthConfigTemplateLoader`, `spi.AuthTypeDetector` , `spi.HostExtractor`. 

I (think) I can just make it exclude all from `spi` which is probably best.

Reply:
[78a3476](https://github.com/maistra/odh-platform/pull/29/commits/78a34762312b605110edef9cbc1b3a4cb8516081)